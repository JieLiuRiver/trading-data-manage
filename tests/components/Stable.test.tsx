import React from 'react';
import { render } from '@testing-library/react';
import StableIcon from '@/components/SvgIcons/Stable';

describe('StableIcon', () => {
  it('should render correctly with default props', () => {
    const { container } = render(<StableIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });

  it('should render correctly with size and color props', () => {
    const { container } = render(<StableIcon size={48} color="gray" />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
