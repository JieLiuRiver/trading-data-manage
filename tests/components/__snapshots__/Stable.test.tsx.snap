// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StableIcon should render correctly with default props 1`] = `
<svg
  class="icon"
  height="24"
  p-id="9429"
  version="1.1"
  viewBox="0 0 1024 1024"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M63.6 489.6h896.7v44.8H63.6z"
    p-id="9430"
  />
</svg>
`;

exports[`StableIcon should render correctly with size and color props 1`] = `
<svg
  class="icon"
  fill="gray"
  height="48"
  p-id="9429"
  version="1.1"
  viewBox="0 0 1024 1024"
  width="48"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M63.6 489.6h896.7v44.8H63.6z"
    p-id="9430"
  />
</svg>
`;

exports[`UpIcon should render correctly with default props 1`] = `
<svg
  class="icon"
  height="24"
  p-id="9429"
  version="1.1"
  viewBox="0 0 1024 1024"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M63.6 489.6h896.7v44.8H63.6z"
    p-id="9430"
  />
</svg>
`;

exports[`UpIcon should render correctly with size and color props 1`] = `
<svg
  class="icon"
  fill="red"
  height="48"
  p-id="9429"
  version="1.1"
  viewBox="0 0 1024 1024"
  width="48"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M63.6 489.6h896.7v44.8H63.6z"
    p-id="9430"
  />
</svg>
`;
